src/Network/ABCI/Types/Error.hs

```diff
@@ -0,0 +1,51 @@
+module Network.ABCI.Types.Error where
+
+import qualified Data.ByteString         as BS
+import qualified Data.ByteString.Base16  as BS16
+import qualified Data.ProtoLens          as PL
+import           Data.String.Conversions (cs)
+
+data Error
+  = CanNotDecodeRequest
+      { canNotDecodeRequestRequest :: BS.ByteString
```

Comment:
it's ugly, let me know if we can do better here

Reply:
seems fine to me